Misc/NEWS.d/next/Tools-Demos/2018-03-16-17-25-05.bpo-29673.m8QtaW.rst

```diff
@@ -0,0 +1,20 @@
+Fix gdbinit
```

Comment:
Thanks for including a NEWS entry.  However, this is a bit too detailed.  See a [current changelog](https://docs.python.org/3.7/whatsnew/changelog.html#changelog) to get a sense of the level of detail we use.